src/main/java/com/google/research/bleth/simulator/Schema.java

```diff
@@ -31,4 +31,15 @@
         public static final String rowNum = "rowNum";
         public static final String colNum = "colNum";
     }
+
+    /** A class providing a single access point for the StatsState entity schema. */
+    public static class StatisticsState {
```

Comment:
Can we store the names of the statistical measures (i.e max, min, avg, etc) as a static class of `Schema` as well?